_posts/pl/2021-02-10-przyklad-migracji-do-chmury.md

```diff
@@ -0,0 +1,888 @@
+---
+layout:    post
+title:     "Migracja aplikacji do Google Cloud w praktyce"
```

Comment:
dobrze rokujący tytuł, jednocześnie jak widzę dosyć często używasz słowa migracj*, ale przydałoby się je rozszerzyć o "do Google Cloud", które w tekście występuje zaledwie 7x i rzadko w towarzystwie migracji. A rzecz w tym, że artykuł jest sporych rozmiarów. Wiadomo, nie chodzi o to żeby teraz przesadzić i wciskać wszędzie, ale jeżeli wcisnąłbyś jeszcze z 5-6x to efekt pozycjonowania zakładam że będzie dużo lepszy.

Reply:
Nie widzę dobrych miejsc do wciśnięcia, żeby nie wyszło sztucznie.
Zmienię tytuł na:
"Migracja aplikacji do chmury Google Cloud Platform w praktyce"
i dodam słowa takie jak "do Google Cloud Platform" tam gdzie pasuje, niestety nie będzie tego tyle ile byś chciał - ale nie widzę już gdzie to wcisnąć.